cmd/create/ocmrole/cmd.go

```diff
@@ -332,7 +336,7 @@ func run(cmd *cobra.Command, argv []string) {
 }
 
 func buildCommands(prefix string, roleName string, rolePath string, permissionsBoundary string,
-	accountID string, env string, isAdmin bool, managedPolicies bool,
+	accountID string, env string, isAdmin bool, managedPolicies bool, autoConfirmLink pflag.Value,
```

Comment:
```suggestion
	accountID string, env string, isAdmin bool, managedPolicies bool, autoConfirmLink bool,
```
Would suggest to bump this to a bool instead of a flag